### PR TITLE
Added sample demonstrating how to connect session with Blazor and WebForms

### PR DESCRIPTION
**Pull request by pockets3407**
- [x] You've read the [Contributor Guide](../CONTRIBUTING.md) and [Code of Conduct](../CODE-OF-CONDUCT.md).
- [x] You've included unit tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

**PR Title**
Updated sample demonstrating how to connect session with Blazor and WebForms using a Blazor Web App

**PR Description**
Updated sample demonstrating how to connect session with Blazor and WebForms using a Blazor Web App

Addresses #382

### CODE REVIEW

**Comment by pockets3407** [comment]
@twsouthwick I redid the branch because I screwed up the rebase yesterday. It should be all set.

File: samples/WebFormsToBlazor/BlazorCore/Program.cs [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
+/*
+ * When Blazor interactive SSR establishes a SignalR connection, a CONNECT method is requested.
+ * This request will stay open until Blazor sends a disconnect request. 
+ * Since it stays open, the session is not properly committed and will stay in the heartbeat loop on the framework side of System.WebAdapters.
+ * We can either:
+ *     A: Block all connect requests from using System.Web.Adapters
+ *     B: Block all requests from paths containing "/_blazor"
+ * TLDR: We need to ensure that SystemWeb.Adapters is not used with Blazor SignalR
+ */
+app.UseWhen(
```
Comment (twsouthwick) [inline samples/WebFormsToBlazor/BlazorCore/Program.cs:45]: Is there a tracking issue for this? Not sure if this is the best way, but we should have an out of the box way to handle this I think.

File: samples/WebFormsToBlazor/BlazorCore/Program.cs [hunk 2]
```diff
@@ -1,27 +1,62 @@
...
+    (context) => HttpMethods.IsConnect(context.Request.Method) == false,
+    appBuilder => appBuilder.UseSystemWebAdapters());
+//app.UseWhen(
+//    (context) => context.Request.Path.ToString().Contains("/_blazor", StringComparison.OrdinalIgnoreCase) == false,
+//    appBuilder => appBuilder.UseSystemWebAdapters());
 
-app.MapBlazorHub();
-app.MapBlazorPages("/_Host");
-app.MapForwarder("/{**catch-all}", app.Configuration["ProxyTo"]!).WithOrder(int.MaxValue);
+app.MapRazorComponents<App>()
```
Comment (twsouthwick) [inline samples/WebFormsToBlazor/BlazorCore/Program.cs:52]: This breaks the existing sample that is to show using blazor components in webforms. It seems like this sample you have should be a separate one

**Review by twsouthwick — CHANGES_REQUESTED** [review]
LGTM, except let's break it out into its own sample so the blazor components in webforms one still works

File: samples/WebFormsToBlazor/BlazorCore/Program.cs (hunk 2 above)
Comment (pockets3407) [reply samples/WebFormsToBlazor/BlazorCore/Program.cs:52]: The existing sample still works. See the images below. The blazor web page is now the index and I have updated the nav bar to reference web form pages. 

image
 
image

I can make a separate sample if needed. I was under the impression from my previous PR that I should combine them.

File: samples/WebFormsToBlazor/BlazorCore/Program.cs (hunk 1 above)
Comment (pockets3407) [reply samples/WebFormsToBlazor/BlazorCore/Program.cs:45]: Not that I know of. I mentioned the deadlock in my previous PR and ended up with this workaround to get my project working.

Comment (twsouthwick) [reply samples/WebFormsToBlazor/BlazorCore/Program.cs:45]: sounds good for the sample for now. could you create an issue to track this?

File: samples/WebFormsToBlazor/BlazorCore/Program.cs (hunk 2 above)
Comment (twsouthwick) [reply samples/WebFormsToBlazor/BlazorCore/Program.cs:52]: awesome! I did not see that initially. it's fine as-is

**Review by twsouthwick — APPROVED** [review]
LGTM - thanks for rebasing and the work here.

File: samples/WebFormsToBlazor/BlazorCore/Program.cs (hunk 1 above)
Comment (pockets3407) [reply samples/WebFormsToBlazor/BlazorCore/Program.cs:45]: #543

**Pull request merged**